### PR TITLE
WIP: azure-sdk-for-go upgrade assessment

### PR DESCRIPTION
**Pull request by m1kola**
This is not for code review, at the moment. Just for me to do some testing.

```release-note
NONE
```

### CODE REVIEW

File: pkg/arm/v9/resources.go [hunk 1]
```diff
@@ -51,8 +52,8 @@ const (
 	vmssAdminUsername                             = "cloud-user"
 )
 
-func (g *simpleGenerator) vnet() *network.VirtualNetwork {
```
Comment (mjudeikis) [inline pkg/arm/v9/resources.go]: Im not sure I am big fan of putting all objects under one package.
I think these could go under `https://github.com/openshift/openshift-azure/tree/master/pkg/util/azureclient` individual subpackages, replicating SDK structure.

Comment (m1kola) [reply pkg/arm/v9/resources.go]: @mjudeikis this is not ready for review: it's a very dirty PR :) I'm looking into what we need to change in our code base to be able to upgrade to the latest version of SDK and this PR is more like notes for me.

Comment (m1kola) [reply pkg/arm/v9/resources.go]: I mean, I'll be happy to rearrange it later, if we decide to go this way at the end.

**Comment by ehashman** [comment]
Closing as stale.

**Pull request closed without merging**